gamification/tests/test_models.py

```diff
@@ -341,3 +347,18 @@ def test_add_badges_method(self):
 
         # After applying add_badge method
         self.assertEquals(test_participant.badges.count(), 0)
+
+    def test_update_participants_data_with_meta_review(self):
+        # Test the participant who has completed the meta-review activity
+        meta_review_participant.objects.create(login='test2')
+        test_meta_review_participant = meta_review_participant.objects.get(
```

Comment:
I have an issue here. Here's what I try to do:

1. Create two meta-review participants. One with meta-review completed while the other has not.
2. Pass both of them through ```update_participants_data_with_meta_review()``` and update.
3. The one with meta-review completed would have added one activity unlike the other.


Now the issue is that when I'm creating the user through ```meta_review_participant```, I'm unable to get the activities count since it's a part of the ```gamification```.

Reply:
Besides creating a meta-review participant, you also need to create a **corresponding** gamification participant.

Reply:
I tried to create a ```gamification``` participant with the same name as that of ```meta-review``` one. Now, the problem is how to link them. 
Though the activity is being added in ```meta-review```, how can I update it in ```gamification``` because in gamification, it's still showing activity count as 0 which in ideal situations should be 0?

Reply:
> Though the activity is being added in meta-review

You only need to play around with meta-review app to fetch some data you need. All the other stuff are related to `gamification`. `Activity` has nothing to do with `meta-review participant`.